crates/uv-trampoline-builder/src/lib.rs

```diff
@@ -325,16 +351,28 @@ pub fn windows_script_launcher(
     let python = python_executable.as_ref();
     let python_path = python.simplified_display().to_string();
 
-    let mut launcher: Vec<u8> = Vec::with_capacity(launcher_bin.len() + payload.len());
-    launcher.extend_from_slice(launcher_bin);
-    launcher.extend_from_slice(&payload);
-    launcher.extend_from_slice(python_path.as_bytes());
-    launcher.extend_from_slice(
-        &u32::try_from(python_path.len())
-            .expect("file path should be smaller than 4GB")
-            .to_le_bytes(),
-    );
-    launcher.extend_from_slice(LauncherKind::Script.magic_number());
+    // Start with base launcher binary
+    // Create temporary file for the launcher
+    let temp_dir = tempfile::tempdir()?;
```

Comment:
```suggestion
    let temp_dir = tempfile::TempDir::new()?;
```

nit: consistency w/ other uv crates